stack/core/src/main/java/org/apache/usergrid/corepersistence/CpEntityManager.java

```diff
@@ -729,6 +740,43 @@ public void createApplicationCollection( String entityType ) throws Exception {
         create( entityType, null );
     }
 
+    @Override
+    public Entity getAllEntityFromFields( String collectionType, String aliasType ){
+        String collName = Schema.defaultCollectionName( collectionType );
+
+        CollectionScope collectionScope = getCollectionScopeNameFromEntityType(
+            getApplicationScope().getApplication(), collName);
+
+        String propertyName = Schema.getDefaultSchema().aliasProperty( collName );
```

Comment:
The method that this replaces to replace never gets fed a list of entity properties, so I didn't feel the need to account for multiple and change the calls. Really it is just getting a collectionType and an alias type because it aims to replace the getAlias method.